### PR TITLE
Added README file

### PR DESCRIPTION
**Pull request by KristyReid**
Adding missing README.md file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Initial readme file
```
Comment (Krystiantt) [inline README.md:1]: It should be more descriptive.

**Review by Krystiantt — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (KristyReid) [reply README.md]: Added description of the project.

**Review by KristyReid — COMMENTED** [review]
Now it's ok.

**Review by Krystiantt — APPROVED** [review]
It's ok now.

**Pull request merged**